web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -51,6 +53,8 @@ jest.mock("@/lib/api-client/apiClient", () => ({
   searchBusinessName: jest.fn(),
 }));
 
+const corporateLegalStructures = ["c-corporation", "s-corporation"];
```

Comment:
[sand] There are couple other test files that have an array of these two legal structures, maybe we could potentially pull it all into a shared file of some sort? Just a thought for a follow up

Reply:
Ya